File: Haskell/beginner/1144.hs
-- https://judge.beecrowd.com/pt/problems/view/1144
-- Sequência Lógica
main :: IO ()
main = do
    -- Reading Usr Input
    usrInput <- readLn :: IO Int

    -- Iterating and Printing
    mapM_ printResult [1..usrInput]
  where
    -- Function to do the job
    printResult i = do
        let square = i * i
        let cube = i * i * i
        putStrLn $ unwords [show i, show square, show cube]
        putStrLn $ unwords [show i, show (square + 1), show (cube + 1)]

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1144.c
// https://judge.beecrowd.com/pt/problems/view/1144
// Sequência Lógica

// Libraries
#include <stdio.h>

int main() {
    // Starting Code
    int usr_input;

    // Scanning user input
    scanf("%d", &usr_input);

    // Iterating
    for (int i = 1; i <= usr_input; i++) {
        printf("%d %d %d\n%d %d %d\n", i, i * i, i * i * i, i, i * i + 1, i * i * i + 1);
    }


    // Ending Code
    return 0;    
}

// By Fernando Serra
// https://github.com/fernandosserra